File: leetcode/questions/2924/csharp/2924.cs
public class Solution {
    public int FindChampion(int n, int[][] edges) {
        var strongests = new HashSet<int>(Enumerable.Range(0, n));
        
        foreach (var edge in edges) {
            strongests.Remove(edge[1]);
        }

        return strongests.Count == 1 ? strongests.ElementAt(0) : -1;
    }
}

File: leetcode/questions/2924/java/2924.java
import java.util.HashSet;
import java.util.Set;

class Solution {
    public int findChampion(int n, int[][] edges) {
        Set<Integer> strongests = new HashSet<>();

        for (int node = 0; node < n; ++node) {
            strongests.add(node);
        }

        for (int[] edge: edges) {
            strongests.remove(edge[1]);
        }

        return strongests.size() == 1 ? strongests.iterator().next() : -1;
    }
¦}